cx/spark/server.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * To-Do:
+ * Create a Service Account and download its key file. 
+ * Set the environmental variable GOOGLE_APPLICATION_CREDENTIALS 
+ * to the key file's location.
+ * See https://dialogflow.com/docs/reference/v2-auth-setup and 
```

Comment:
This link seems to redirect users to a page which is not related to service account setup. Probably try to find and use the corresponding guide in latest CX doc.